app/addClothe.tsx

```diff
@@ -1,59 +1,242 @@
-import {Button, StyleSheet, TextInput} from 'react-native';
+import {Button, StyleSheet, TextInput, Image, Text, Platform} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
 import {View} from '@/components/Themed';
 import {useState} from "react";
 import {useNavigation} from "@react-navigation/native";
 import DefaultClothe from "@/classes/clothe";
 import {DatabaseController} from "@/classes/DatabaseController";
+import Filter from '@/components/Filter';
+import { withTheme } from 'react-native-elements';
+import { Clothe } from '@/classes/clothe';
 
 export default function AddClotheScreen() {
     const [clothe] = useState(DefaultClothe());
     const navigation = useNavigation();
-
-    // Adds clothe to DB and changes screen back to the previous one
+    const [selectedLanguage, setSelectedLanguage] = useState();
+    const [selectedName, setSelectedName] = useState("Clothe Item");
+    const [selectedDescription, setSelectedDescription] = useState("");
+    const [selectedLink, setSelectedLink] = useState(clothe.link);
+    const [selectedType, setSelectedType] = useState(Clothe.Type.Unknown);
+    const [selectedColor, setSelectedColor] = useState(""); // Default value
+    const [selectedSize, setSelectedSize] = useState(-1); // Default value
+    
+    
+    const handleCancel = () => {
```

Comment:
I think there should be more consistency here, we should either use the arrow function expression or the regular function declaration, not both.

Seeing as there was a function already written using a regular function declaration (`addClothe`), I think we should stick with that.